### ISSUE TITLE
Docs link doesn't seem to be anywhere

### ISSUE THREAD

**Issue by dhermes** [jameskermode/f90wrap#69]
Are your docs hosted somewhere? ISTM you have Sphinx docs ready to go (I'll clone this and check). I'm happy to show you how to host them on https://readthedocs.org/ if you'd like.

**Comment by jameskermode** [comment]
Thanks for the offer, a PR to do this would be appreciated. However, the problem is more that the docs are not up to date and complete. The `README` should be a good starting point, together with the collection of examples/tests.

**Comment by dhermes** [comment]
Indeed. I tried building the docs and noticed references to defunct docs. Do you know if there is a commit in history where the docs actually built successfully?

**Comment by jameskermode** [comment]
Closing with preference to #293
